cacti/README.md

```diff
@@ -9,109 +9,157 @@ Get metrics from cacti service in real time to:
+
 ```shell
 sudo -u dd-agent /opt/datadog-agent/embedded/bin/pip install rrdtool
 ```
 
 ### Configuration
 
-Create a datadog user with read-only rights to the Cacti database
+#### Create a Datadog User
 
-```shell
-sudo mysql -e "create user 'datadog'@'localhost' identified by '<password>';"
-sudo mysql -e "grant select on cacti.* to 'datadog'@'localhost';"
-```
+1. Create a datadog user with read-only rights to the Cacti database
 
-Check user and rights
+    ```shell
+    sudo mysql -e "create user 'datadog'@'localhost' identified by '<MYSQL_PASSWORD>';"
+    sudo mysql -e "grant select on cacti.* to 'datadog'@'localhost';"
+    ```
 
-```shell
-mysql -u datadog --password=<password> -e "show status" | \
-grep Uptime && echo -e "\033[0;32mMySQL user - OK\033[0m" || \
-echo -e "\033[0;31mCannot connect to MySQL\033[0m"
+2. Check user and rights:
 
-mysql -u datadog --password=<password> -D cacti -e "select * from data_template_data limit 1" && \
-echo -e "\033[0;32mMySQL grant - OK\033[0m" || \
-echo -e "\033[0;31mMissing SELECT grant\033[0m"
-```
+    ```shell
+    mysql -u datadog --password=<MYSQL_PASSWORD> -e "show status" | \
+    grep Uptime && echo -e "\033[0;32mMySQL user - OK\033[0m" || \
+    echo -e "\033[0;31mCannot connect to MySQL\033[0m"
 
-Configure the Agent to connect to MySQL, edit your `cacti.d/conf.yaml` file. See the [sample cacti.d/conf.yaml][3] for all available configuration options:
-
-```yaml
-init_config:
-
-instances:
-    -   mysql_host: localhost
-        mysql_user: datadog
-        mysql_password: hx3beOpMFcvxn9gXcs0MU3jX
-        rrd_path: /path/to/cacti/rra
-        #field_names:
-        #    - ifName
-        #    - dskDevice
-        #    - ifIndex
-        #rrd_whitelist: /path/to/rrd_whitelist.txt
-```
+    mysql -u datadog --password=<MYSQL_PASSWORD> -D cacti -e "select * from data_template_data limit 1" && \
+    echo -e "\033[0;32mMySQL grant - OK\033[0m" || \
+    echo -e "\033[0;31mMissing SELECT grant\033[0m"
+    ```
 
-Give the datadog-agent user access to the RRD files
+3. Give the datadog-agent user access to the RRD files:
 
-```shell
-sudo gpasswd -a dd-agent www-data
-sudo chmod -R g+rx /var/lib/cacti/rra/
-sudo su - datadog-agent -c 'if [ -r /var/lib/cacti/rra/ ];
-then echo -e "\033[0;31mdatadog-agent can read the RRD files\033[0m";
-else echo -e "\033[0;31mdatadog-agent can not read the RRD files\033[0m";
-fi'
-```
+    ```shell
+    sudo gpasswd -a dd-agent www-data
+    sudo chmod -R g+rx /var/lib/cacti/rra/
+    sudo su - datadog-agent -c 'if [ -r /var/lib/cacti/rra/ ];
+    then echo -e "\033[0;31mdatadog-agent can read the RRD files\033[0m";
+    else echo -e "\033[0;31mdatadog-agent can not read the RRD files\033[0m";
+    fi'
+    ```
+
+4. Follow the instructions below to configure this check for an Agent running on a host. For containerized environments, see the [Containerized](#containerized) section.
```

Comment:
Move this sentence below `#### Host` to stay consistent with other integration pages.